snippets/messaging/cloud_messaging.py

```diff
@@ -35,7 +37,7 @@ def send_to_token():
     # registration token.
     response = messaging.send(message)
     # Response is a message ID string.
-    print('Successfully sent message:', response)
+    print(('Successfully sent message:', response))
```

Comment:
This is going to print out a tuple, which is not what we want. 

```
>>> print(('foo', 'bar'))
('foo', 'bar')
```

Please remove the redundant parenthesis -- here and below. Then I can merge this PR.